tests/kubectl_test.go

```diff
@@ -0,0 +1,69 @@
+package tests_test
+
+import (
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("[rfe_id:3423][vendor:cnv-qe@redhat.com][level:component]oc/kubectl get vm/vmi tests", func() {
+	tests.FlagParse()
+
+	var k8sClient string
+
+	virtCli, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+
+	BeforeEach(func() {
+		k8sClient = tests.GetK8sCmdClient()
+		tests.SkipIfNoCmd(k8sClient)
+		tests.BeforeTestCleanup()
+	})
+
+	table.DescribeTable("should verify set of columns for", func(verb, resource string, expectedHeader []string) {
+		vm := tests.NewRandomVirtualMachine(tests.NewRandomVMI(), false)
+		vm, err := virtCli.VirtualMachine(tests.NamespaceTestDefault).Create(vm)
+		Expect(err).NotTo(HaveOccurred())
+
+		tests.StartVirtualMachine(vm)
+
+		result, _, _ := tests.RunCommand(k8sClient, verb, resource)
+		Expect(result).ToNot(BeNil())
+		resultFields := strings.Fields(result)
+		columnHeaders := resultFields[:len(expectedHeader)]
+		// Verify the generated header is same as expected
+		Expect(columnHeaders).To(Equal(expectedHeader))
+		// Name will be there in all the cases, so verify name
+		Expect(resultFields[len(expectedHeader)]).To(Equal(vm.Name))
+	},
+		table.Entry("virtualmachine", "get", "vm", []string{"NAME", "AGE", "RUNNING", "VOLUME"}),
```

Comment:
test id: CNV-3421

Reply:
@ILpinto those are already mentioned here https://github.com/kubevirt/kubevirt/pull/2937/files#diff-c777d78c8b521c23b8cada997b5ebed6R66-R67

The first two are without `-o wide`